src/main/resources/extensions/settings.yaml

```
@@ -31,3 +31,38 @@ spec:
           key: withReplySize
           validation: required
           value: 5
+    - group: avatar
+      label: 头像设置
+      formSchema:
+        - $formkit: checkbox
+          label: 是否启用第三方头像
+          name: enable
+          id: enable
+          key: enable
+          value: false
+        - $formkit: select
+          label: 头像服务提供者
+          if: "$get(enable).value === true"
+          name: provider
+          options:
+            - label: 'Gravatar'
+              value: 'Gravatar'
+          value: "Gravatar"
+        - $formkit: text
+          label: 头像服务镜像地址
+          if: "$get(enable).value === true"
+          name: providerMirror
+          value: ""
+          help: 使用官方源则留空
+        - $formkit: select
+          label: 头像策略
+          if: "$get(enable).value === true"
+          name: policy
+          options:
+            - label: '仅匿名用户'
+              value: 'AnonymousUserPolicy'
```

```suggestion
              value: 'anonymousUserPolicy'
```

同上。另外，字段名已经是 `policy`，那么 value 我觉得可以忽略掉 `Policy`，直接使用 `anonymousUser`。